EasyIntern-Backend/Controllers/AccountController.cs

```diff
@@ -111,20 +111,28 @@ public async Task<IActionResult> RegisterCompany([FromBody] JsonModelRegisterCom
 
         _context.Users.Add(user);
         await _context.SaveChangesAsync();
-        return Ok();
+        // Return the user object
+        // TODO: convert this to something more portable
+        return Json(new  {
+            id = user.Id,
+            name = user.Name,
+            email = user.Email,
+            userType = user.UserType,
+            profileSettings = user.ProfileSettings.ToDictionary(e => e.Key, e => e.Value)
+        });
```

Comment:
You can safely use pascal case here, ASP.NET automatically converts every json property to camel case. This also allows you to use auto-properties ('new { user.Id }' instead of 'new { Id = user.Id }')

Reply:
Fixe